### ISSUE TITLE
possible to remove rb-appscript dependency?

### ISSUE THREAD

**Issue by jacquescrocker** [achiurizo/consular-osx#1]
rb-appscript looks like a dead project, and it won't install on my system anymore :(

Is there any other way to get consular to work?

growl_notify author recommends against (rb-appscript), see description of https://github.com/scottdavis/growl_notify

**Comment by achiurizo** [comment]
Yeah looks like they stopped: http://appscript.sourceforge.net/status.html. Thanks for bringing this to my attention. Unfortunately `consular-osx`(and `consular-iterm`) uses `rb-appscript` so I will search up another alternative to get this working as soon as possible.

**Comment by thewoolleyman** [comment]
This is going to keep biting more and more people, and eventually make consular unusable.  Until you can fix it correctly by removing the dependency, why don't you just fork the rb-appscript gem with a new name (as described in https://github.com/achiu/consular-osx/issues/7), publish it, and depend on that?

Or alternately, if the rb-appscript admins let you, just get admin access patch the original repo and publish a patch version to rubygems.

**Comment by cmbirk** [comment]
Bumping this, as I'm currently running into the same issue.  Has there been any progress on this issue?

**Comment by cmbirk** [comment]
I know `consular-iterm` installation works.  Can we port the changes from there to `consular-osx`?
